testenv/README.md

```diff
@@ -34,11 +34,20 @@ You can customize these in
 3. bootbox/puppet/modules/genesis/templates/config.yaml.erb
 4. bootbox/puppet/modules/genesis/templates/stage2.erb.
 
-and modify settings as desired.
+and modify settings as desired. While the current values of the options
+allow you to proceed in the test environment without updates by executing
+```vagrant up```, you MUST provide production settings for
+
+* a DHCP server,
+* a TFTP server,
+* and a file server (serving static files over HTTP)
+* RPM repository that includes RPMs required by Genesis tasks
```

Comment:
I think this is harmless and probably a good idea to repeat. Leaving this in here for now